blockauth/apple/notification_service.py

```
@@ -55,6 +62,74 @@ def _parse_events_claim(raw: Any) -> dict[str, Any]:
+    if event_time > 9_999_999_999:
+        event_time = event_time / 1000
+    if not math.isfinite(event_time):
+        raise AppleNotificationVerificationFailed("Apple notification event_time is not finite")
+    return event_time
+
+
+def _int_setting(key: str, default: int, *, minimum: int) -> int:
+    try:
+        value = int(apple_setting(key, default))
+    except (TypeError, ValueError) as exc:
+        raise AppleNotificationVerificationFailed(f"{key} must be an integer") from exc
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)